[2025-03-16T20:12:25.020+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: passanger_traffic.creare1 manual__2025-03-16T20:12:17.809278+00:00 [queued]>
[2025-03-16T20:12:25.025+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: passanger_traffic.creare1 manual__2025-03-16T20:12:17.809278+00:00 [queued]>
[2025-03-16T20:12:25.026+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-03-16T20:12:25.032+0000] {taskinstance.py:1380} INFO - Executing <Task(PostgresOperator): creare1> on 2025-03-16 20:12:17.809278+00:00
[2025-03-16T20:12:25.035+0000] {standard_task_runner.py:57} INFO - Started process 197 to run task
[2025-03-16T20:12:25.036+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'passanger_traffic', 'creare1', 'manual__2025-03-16T20:12:17.809278+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/passenger_traffic_dag.py', '--cfg-path', '/tmp/tmp_caamzfb']
[2025-03-16T20:12:25.038+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask creare1
[2025-03-16T20:12:25.047+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-16T20:12:25.070+0000] {task_command.py:415} INFO - Running <TaskInstance: passanger_traffic.creare1 manual__2025-03-16T20:12:17.809278+00:00 [running]> on host b70acf7f0e92
[2025-03-16T20:12:25.123+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='passanger_traffic' AIRFLOW_CTX_TASK_ID='creare1' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T20:12:17.809278+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T20:12:17.809278+00:00'
[2025-03-16T20:12:25.125+0000] {sql.py:274} INFO - Executing: 
    CREATE SCHEMA IF NOT EXISTS presentation;
[2025-03-16T20:12:25.131+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:12:25.136+0000] {base.py:73} INFO - Using connection ID '***_dwh' for task execution.
[2025-03-16T20:12:25.141+0000] {sql.py:418} INFO - Running statement: 
    CREATE SCHEMA IF NOT EXISTS presentation;, parameters: None
[2025-03-16T20:12:25.148+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=passanger_traffic, task_id=creare1, execution_date=20250316T201217, start_date=20250316T201225, end_date=20250316T201225
[2025-03-16T20:12:25.175+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-16T20:12:25.192+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
